app/serializers/web_hook_user_serializer.rb

```diff
@@ -30,6 +30,7 @@ def staff_attributes(*attrs)
     can_change_bio
     can_change_location
     can_change_website
+    can_change_tracking_preferences
```

Comment:
I don't think we should include this in the user web hook yet as this is doesn't really provide any benefit for the webhook.

Reply:
Actually this code *was* removing it from the serializer. It is excluding attrs from the base serializer. I'll put it back.